Identify RTM-LAMP temperature and voltage sensors

joao-victor-s opened the pull request:
Align `attr` fields with the SDR structures of the RTM-LAMP LM75 temperature sensors to indicate which part of the card is being monitored

Also add support for MAX11609-based housekeeping voltage sensors on the RTM-LAMP

gustavosr8 commented on ipmiMgrApp/Db/rtm_lamp.substitutions:
```diff
@@ -24,7 +33,15 @@ file "fru_common_alias.db"
-            { TempRTM1, 1       , 1   , $(fruid), alias_$(dev), $(dev) }
-            { TempRTM2, 2       , 1   , $(fruid), alias_$(dev), $(dev) }
-            { TempRTM3, 3       , 1   , $(fruid), alias_$(dev), $(dev) }
+            { TempAirIn, 1  , 1   , $(fruid), alias_$(dev), $(dev) }
+            { TempAirOut, 2 , 1   , $(fruid), alias_$(dev), $(dev) }
+            { TempPSBlock, 3, 1   , $(fruid), alias_$(dev), $(dev) }
+            { VoltVS1, 1    , 2   , $(fruid), alias_$(dev), $(dev) }
+            { VoltVS2, 2    , 2   , $(fruid), alias_$(dev), $(dev) }
+            { VoltN7V, 3    , 2   , $(fruid), alias_$(dev), $(dev) }
+            { Volt7V,  4    , 2   , $(fruid), alias_$(dev), $(dev) }
+            { Volt2V5, 5    , 2   , $(fruid), alias_$(dev), $(dev) }
+            { Volt3V3, 6    , 2   , $(fruid), alias_$(dev), $(dev) }
+            { Volt5V,  7    , 2   , $(fruid), alias_$(dev), $(dev) }
+            { Volt12V_HP, 8 , 2   , $(fruid), alias_$(dev), $(dev) }
```
Please, keep the items aligned in the `attr` column.

```suggestion
            { TempAirIn,   1, 1   , $(fruid), alias_$(dev), $(dev) }
            { TempAirOut,  2, 1   , $(fruid), alias_$(dev), $(dev) }
            { TempPSBlock, 3, 1   , $(fruid), alias_$(dev), $(dev) }
            { VoltVS1,     1, 2   , $(fruid), alias_$(dev), $(dev) }
            { VoltVS2,     2, 2   , $(fruid), alias_$(dev), $(dev) }
            { VoltN7V,     3, 2   , $(fruid), alias_$(dev), $(dev) }
            { Volt7V,      4, 2   , $(fruid), alias_$(dev), $(dev) }
            { Volt2V5,     5, 2   , $(fruid), alias_$(dev), $(dev) }
            { Volt3V3,     6, 2   , $(fruid), alias_$(dev), $(dev) }
            { Volt5V,      7, 2   , $(fruid), alias_$(dev), $(dev) }
            { Volt12V_HP,  8, 2   , $(fruid), alias_$(dev), $(dev) }

```

gustavosr8 commented on ipmiMgrApp/Db/rtm_lamp.substitutions:
```diff
@@ -7,14 +7,23 @@
+# 36   Compact  0xf0    0xc0  0x62  0x10              HS 091 RTM2
 #------------------------------------------------------------------
 
+
```
nit: extra empty line

gustavosr8 reviewed (commented):
Also, please update the [PVList](https://github.com/lnls-dig/ipmi-mgr-epics-ioc/blob/master/PVList.md) with the new PVs.

gustavosr8 commented:
This changes refers to a specif (yet unreleased) openMMC version and only should be merged when the new release comes up, IMO. The commit message should also refer to it. Maybe something like:

> Identify RTM-LAMP sensors following openMMC vx.x.x.
> 
> Align the `attr` field with the SDR structures of the RTM-LAMP. The LM75
> temperature sensors now indicate which part of the card is being
> monitored, and support has been added for the MAX110609 housekeeping
> voltage sensor.
>

gustavosr8 committed:
Update RTM-LAMP sensors following openMMC v1.6.2.

Synchronize the attr field with the SDR structures of the RTM-LAMP. The
temperature sensors now indicate which part of the module is being
monitored, and support has been added for the voltage sensor.
